lib/charms/operator_libs_linux/v2/snap.py

```diff
@@ -937,7 +957,7 @@ def remove(snap_names: Union[str, List[str]]) -> Union[Snap, List[Snap]]:
     if not snap_names:
         raise TypeError("Expected at least one snap to add, received zero!")
 
-    return _wrap_snap_operations(snap_names, SnapState.Absent, "", False)
+    return _wrap_snap_operations(snap_names, SnapState.Absent, "", False, False)
```

Comment:
Ick, another case where I think we should be using keyword arguments -- it's even more clear here with "False, False". Can we change these call sites to use keyword arguments too, while we're at it?

Reply:
Done